lib/fortnox/api/validators/customer.rb

```diff
@@ -1,22 +1,25 @@
 require "fortnox/api/validators/base"
+require "fortnox/api/validators/attributes/country_code"
+require "fortnox/api/validators/attributes/currency"
 
 module Fortnox
   module API
     module Validator
       class Customer < Fortnox::API::Validator::Base
 
+        include Fortnox::API::Validator::Attribute::CountryCode
+        include Fortnox::API::Validator::Attribute::Currency
+
         using_validations do
 
           VAT_TYPES = ['SEVAT', 'SEREVERSEDVAT', 'EUREVERSEDVAT', 'EUVAT', 'EXPORT']
+          TYPES = ['PRIVATE', 'COMPANY']
 
           validates_presence_of :name
 
-          validates_length_of :currency,      length: 3,  if: :currency?
-          validates_length_of :country_code,  length: 2,  if: :country_code?
-
-          validates_inclusion_of :sales_account,  within: (0..9999),               if: :sales_account?
-          validates_inclusion_of :type,           within: ['PRIVATE', 'COMPANY'],  if: :type?
-          validates_inclusion_of :vat_type,       within: VAT_TYPES,               if: :vat_type?
+          validates_inclusion_of :sales_account,  within: (0..9999),  if: :sales_account?
```

Comment:
We should really extract account into it's own attribute class. Only problem is how to name it :) We might get the same problem with country code and currency as well, if they aren't always called exactly that. I'll have a look at it, just ignore it for this PR and we'll refactor when I have a solution for it.